### PR TITLE
AMMP-2825: JSON schema for config and data payloads

### PR DESCRIPTION
**Pull request by svet-b**
First iteration of JSON schema

### CODE REVIEW

**Comment by svet-b** [comment]
More info at https://ammpio.atlassian.net/browse/AMMP-2825

File: resources/schema/config.schema.json [hunk 1]
```diff
@@ -0,0 +1,407 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://github.com/ammpio/ammp-edge/blob/main/resources/schama/config.schema.json",
```
Comment (tienthai-ammp) [inline resources/schema/config.schema.json:3]: `"$id": "https://github.com/ammpio/ammp-edge/blob/main/resources/schema/config.schema.json",`

Comment (tienthai-ammp) [inline resources/schema/config.schema.json:3]:
```suggestion
    `"$id": "https://github.com/ammpio/ammp-edge/blob/main/resources/schema/config.schema.json",`
```

File: resources/schema/data.schema.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://github.com/ammpio/ammp-edge/blob/main/resources/schama/data.schema.json",
```
Comment (tienthai-ammp) [inline resources/schema/data.schema.json:3]:
```suggestion
    "$id": "https://github.com/ammpio/ammp-edge/blob/main/resources/schema/data.schema.json",
```

**Pull request merged**